.roo/rules-translate/001-general-rules.md

```diff
@@ -1,6 +1,6 @@
 # 1. SUPPORTED LANGUAGES AND LOCATION
 
-- Localize all strings into the following locale files: ca, de, es, fr, hi, id, it, ja, ko, nl, pl, pt-BR, ru, tr, vi, zh-CN, zh-TW
+- Localize all strings into the following locale files: ca, de, en, es, fr, hi, id, it, ja, ko, nl, pl, pt-BR, ru, tr, vi, zh-CN, zh-TW
```

Comment:
@EamonNerbonne might have an opinion about this

Reply:
Lol. But no. I guess their aim is to allow translations from something into en?